lnwallet/chancloser/rbf_coop_transitions.go

```diff
@@ -483,6 +581,7 @@ func (c *ChannelFlushing) ProcessEvent(event ProtocolEvent, env *Environment,
 		closeTerms := CloseChannelTerms{
 			ShutdownScripts:  c.ShutdownScripts,
 			ShutdownBalances: msg.ShutdownBalances,
+			NonceState:       c.NonceState,
```

Comment:
Q: so they all share the same nonce state here?
https://github.com/lightningnetwork/lnd/blob/85a5bf294c9284706d95d7e44aa7d3ba85633de5/lnwallet/chancloser/rbf_coop_transitions.go#L556-L566